grails-app/migrations/views/changelog.xml

```diff
@@ -75,7 +75,15 @@
     <sqlFile path="views/stocklist-item-list.sql"/>
   </changeSet>
   <changeSet id="1580848680306-23" author="awalkowiak" runOnChange="true" runAlways="true" failOnError="true">
-    <sqlFile path="views/order-summary-view.sql"/>
+    <sqlFile path="views/order-summary-helper-views.sql"/>
```

Comment:
Does this change trigger a checksum validation?

Reply:
No complains found in the console about this